src/overlays/actors/ovl_En_Horse_Ganon/z_en_horse_ganon.c

```diff
@@ -25,29 +44,276 @@ const ActorInit En_Horse_Ganon_InitVars = {
+        { 13, { { 0, 0, 0 }, 20 }, 100 },
+    },
+};
+static ColliderJntSphInit jntsphInit = {
+    { COLTYPE_UNK10, 0x00, 0x09, 0x39, 0x12, COLSHAPE_JNTSPH },
+    1,
+    jntsphItemsInit,
+};
+static CollisionCheckInfoInit collisionCheckInfoInit = {
+    0x0A,
+    0x0023,
+    0x0064,
+    0xFE,
+};
+s8 D_80A69248[] = { 0x09, 0xB8, 0x01, 0x26, 0x0E, 0x2C, 0x07, 0x00, 0x0C, 0x11, 0x01, 0x7A, 0x12, 0x69, 0x07, 0x00,
+                    0x06, 0x4E, 0xFE, 0xFB, 0x1D, 0xAC, 0x07, 0x00, 0x02, 0xF2, 0xFF, 0x45, 0x24, 0x4F, 0x07, 0x00,
+                    0xF9, 0x6E, 0xFE, 0x0C, 0x31, 0x22, 0x07, 0x00, 0xF3, 0x28, 0xFE, 0x0C, 0x32, 0xD5, 0x07, 0x00,
+                    0xEB, 0xEA, 0xFE, 0x5F, 0x2D, 0x6E, 0x07, 0x00, 0xE9, 0x5E, 0xFE, 0x27, 0x25, 0x65, 0x07, 0x00,
+                    0xE5, 0x93, 0xFE, 0x0C, 0x20, 0xAC, 0x07, 0x00, 0xE6, 0x25, 0xFE, 0x77, 0x1B, 0x07, 0x07, 0x00,
+                    0xEB, 0xB7, 0x00, 0x7C, 0x15, 0x39, 0x07, 0x00, 0xF4, 0x66, 0x00, 0x02, 0x11, 0xB9, 0x07, 0x00,
+                    0xF4, 0x7B, 0xFF, 0xDD, 0x11, 0xAF, 0x07, 0x00, 0xF8, 0x8D, 0xFF, 0xD1, 0x0B, 0xA2, 0x07, 0x00 };
+s32 D_80A692B8[] = { 0, 0x00000010 };
+static InitChainEntry initChain[] = {
+    ICHAIN_F32(unk_F8, 1200, ICHAIN_STOP),
+};
+static void (*actionFuncs[])(EnHorseGanon*, GlobalContext*) = { func_80A68AF0, func_80A68DB0, NULL };
+
+const f32 D_80A692D0 = 10430.3779f;
+
+void func_80A68660(Vec3s* data, s32 index, Vec3f* vec) {
+    Vec3s* temp = (Vec3s*)((u64*)data + index);
+
+    vec->x = (f32)temp->x;
+    vec->y = (f32)temp->y;
+    vec->z = (f32)temp->z;
+}
+
+void func_80A686A8(EnHorseGanon* this, GlobalContext* globalCtx) {
+    Vec3f* tempPos;
+    Vec3f vec;
+    s16 y;
+
+    func_80A68660((Vec3s*)D_80A69248, this->unk_1EC, &vec);
+    if (Math3D_Vec3f_DistXYZ(&vec, &this->actor.posRot.pos) <= 400.0f) {
+        this->unk_1EC += 1;
+        if (this->unk_1EC >= 14) {
+            this->unk_1EC = 0;
+            func_80A68660((Vec3s*)D_80A69248, 0, &vec);
+        }
+    }
+
+    tempPos = &this->actor.posRot.pos;
+    y = Math_Vec3f_Yaw(tempPos, &vec) - this->actor.posRot.rot.y;
+    if (y >= 301) {
+        this->actor.posRot.rot.y += 300;
+    } else {
+        if (y < -300) {
+            this->actor.posRot.rot.y -= 300;
+        } else {
+            this->actor.posRot.rot.y += y;
+        }
+    }
+    this->actor.shape.rot.y = this->actor.posRot.rot.y;
+
+    if (func_8002DB8C(&this->actor, globalCtx->actorCtx.actorList[ACTORTYPE_PLAYER].first) <= 300.0f) {
+        if (this->actor.speedXZ < 12.0f) {
+            this->actor.speedXZ += 1.0f;
+        } else {
+            this->actor.speedXZ -= 1.0f;
+        }
+    } else if (this->actor.speedXZ < (f32)((u8*)D_80A69248 + 6)[this->unk_1EC * 8]) {
+        this->actor.speedXZ += 0.5f;
+    } else {
+        this->actor.speedXZ -= 0.5f;
+    }
+}
+
+#ifdef NON_MATCHING
+void func_80A68870(EnHorseGanon* this)
+// regalloc mismatch
+{
```

Comment:
The formatter failed to fix the brace style here because of the comment.
I suggest having non matching comments between the `#ifdef NON_MATCHING` line and the header of the function.